tests/test_policy.py

```diff
@@ -1435,7 +1435,11 @@ def test_config_poll(self):
         requests = []
 
         def record_requests(Evaluations, ResultToken):
-            requests.append(Evaluations)
+            for e in Evaluations:
+                if e['ComplianceResourceId'] == "dev3":
+                    e['ComplianceType'] = 'NOT_APPLICABLE'
+                    e['Annotation'] = 'The rule does not apply.'
+                requests.append(e)
```

Comment:
@kk1532 This doesn't do what we want to do. Instead of doing this, we need to mock out the boto3 method that fetches the old. existing evaluations from config service and have it return a list of evaluations where one of the evaluations is for a resource that no longer exists. Let me go ahead and put together something.